tp1/tp1.go

```diff
@@ -24,17 +41,75 @@ type Carrito struct {
 // para cada super mercado, cuanto saldria comprar esos productos ahi.
 // Retorna un slice de carritos, donde se tiene uno para cada super mercado.
 func (p Productos) CalcularPrecios(ids ...int) []Carrito {
-	return nil
+
+	carritos := []Carrito{}
+
+	listadoDeProductos, _ := LeerProductos("productos.json")
+
+	for _, supermercado := range Supermercados {
```

Comment:
Dentro de `p` ya tenemos cargadas las tiendas que vamos a estar utilizando. Y `p` podria no tener tiendas que si estan en `Supermercados`, al agregar este for vamos a estar iterando en y revisando cosas que no deberiamos. Podriamos simplementar usar `p` para implementarlo y buscar una forma de ir guardando las tiendas que voy visitando